tasks/create_ordered_kafka_groups.yml

```diff
@@ -19,13 +19,13 @@
     owner: "{{kafka_broker_user}}"
     group: "{{kafka_broker_group}}"
   when:
-    - "'zookeeper.ssl.client.enable=true' in slurped_properties.content|b64decode"
```

Comment:
im guessing you have a secret protection enabled cluster, can you confirm?

secrets protection puts spaces in, that we normally do not